build.sbt

```diff
@@ -54,7 +55,8 @@ libraryDependencies := Seq(
   "org.typelevel"              %% "cats-testkit"             % catsVersion % "test",
   "org.typelevel"              %% "cats-effect"              % catsEffectVersion,
   "org.typelevel"              %% "mouse"                    % mouseVersion,
-  )
+  "io.chrisdavenport"          %% "cats-par"                 % catsParVersion,
```

Comment:
This is a temporary hack that will go away with Cats 2.0. The short version is that you can say `F[_]: Par` which then lets you say `fa.parTraverse(...)` which I use for running ITC requests in parallel. Basically it gives you a more useful way to deal with the existing but flawed `Parallel` multi-parameter typeclass. See [here](https://github.com/ChristopherDavenport/cats-par) and the link to the related issue if you want a explanation.